pkg/specgenutil/volumes.go

```diff
@@ -343,6 +343,12 @@ func getBindMount(args []string) (spec.Mount, error) {
 			default:
 				return newMount, errors.Wrapf(util.ErrBadMntOption, "%s mount option must be 'private' or 'shared'", kv[0])
 			}
+		case "U":
+			if setOwnership {
+				return newMount, errors.Wrapf(optionArgError, "cannot pass 'U' option more than once")
+			}
+			setOwnership = true
+			newMount.Options = append(newMount.Options, kv[0])
```

Comment:
This does not handle the `...,U=false` case. You never check what kv[1] is set to. You should validate that it is set to either true or false, right now I can do `...,U=test`